### DISCUSSION TITLE
Introduce more options around working directory inheritance

### DISCUSSION THREAD

**Discussion by zx8** [ghostty-org/ghostty#3183] [Feature Requests, Ideas] [answered]
Coming from iTerm, I have the following setup:

* For split panes, inherit the working directory from the originally-focused pane
* For new windows, use the home directory
* For new tabs, use the home directory

The only options that affect this behaviour at the moment are:

* `working-directory` – specify a static directory
* `window-inherit-working-directory` – opt in/out of inheritance for tabs/panes/windows

Perhaps the following new options could be introduced to control the behaviour at a more granular level:

* `tab-inherit-working-directory`
* `pane-inherit-working-directory`

**Comment by mitchellh** [comment, marked as answer]
Dupe: #1392 already accepted!
